scripts/check-install.ts

```diff
@@ -0,0 +1,28 @@
+import * as path from 'path';
+import { globP, ncpP, spawnP, tmpDirP } from './utils';
+
+/**
+ * This function checks that the following two (sequential) operations succeed:
+ * 1. In a temporary directory, installs from the `npm pack` of this directory
+ * 2. Compiles a top-level file in that directory that imports this module
+ */
+export default async function() {
+  // This script assumes that you don't already have a TGZ file
+  // in your current working directory.
+  const installDir = await tmpDirP();
+  console.log(installDir);
+  await spawnP('npm', ['pack']);
+  const tgz = (await globP(`${process.cwd()}/*.tgz`)).join(' '); // should be only one
```

Comment:
If there should be only one .tgz file, why not programmatically check it?

Reply:
Makes sense, added throwing an error.